src/Geta.EPi.Extensions/LinkItemExtensions.cs

```diff
@@ -45,11 +45,22 @@ public static IContent ToContent(this LinkItem source)
         /// <returns>Returns friendly URL if item is EPiServer content, otherwise returns the original Href property value.</returns>
         public static string GetFriendlyUrl(this LinkItem linkItem, bool includeHost = false)
         {
-            var contentReference = linkItem.ToContentReference();
+            if (linkItem.Href == null)
```

Comment:
Why not checking `string.IsNullOrEmpty'? Is empty string a valid value to proceed?